perceval/backends/core/github.py

```diff
@@ -389,18 +389,41 @@ def __get_pull_requested_reviewers(self, pr_number):
         return requested_reviewers
 
     def __get_pull_commits(self, pr_number):
-        """Get pull request commit hashes"""
+        """Get pull request commit hashes and comments"""
 
-        hashes = []
+        commits = []
         group_pull_commits = self.client.pull_commits(pr_number)
 
         for raw_pull_commits in group_pull_commits:
 
             for commit in json.loads(raw_pull_commits):
-                commit_hash = commit['sha']
-                hashes.append(commit_hash)
+                commit_info = {
+                    'hash': commit['sha'],
+                    'commit_comments': self.__get_commit_comments(commit['sha'])
+                }
 
-        return hashes
+                commits.append(commit_info)
+
+        return commits
+
+    def __get_commit_comments(self, commit_sha):
+        """Get pull request commit comments"""
+
+        comments = []
+        group_comments = self.client.pull_commit_comments(commit_sha)
+
+        for raw_comments in group_comments:
+
+            for comment in json.loads(raw_comments):
```

Comment:
Hey @valeriocos , I checked the coverage report and the lines not covered are line#417 to line#420, and line#425 in the here. I think it is due to the fact that in test_github.py, for the  "zhquan_example/repo", there are no commit comments (the group_comments is an empty list), and so these lines are not executed.
Please let me know what do you think, thanks :)